docs/transfer.md

```diff
@@ -0,0 +1,32 @@
+# Transfer Service
+
+The transfer service is a simple flexible service which can be used to transfer artifact objects between a source and destination. The service determines whether the transfer between the source and destination is possible rather than the API.
```

Comment:
```
The service determines whether the transfer between the source and destination is possible rather than the API.
```

Can we have a little more detailed explanation of what is really meant here?